src/components/DocExplorer/__tests__/TypeDoc-test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
+
+import TypeDoc from '../TypeDoc';
+import {
+  ExampleSchema,
+  ExampleQuery,
+  ExampleUnion,
+  ExampleEnum
+} from '../../__tests__/ExampleSchema'
+
+describe('TypeDoc', () => {
+  it('renders a top-level query object type', () => {
+    const onClickType = jest.fn();
```

Comment:
Would be nice to have an assertion on this.

Reply:
i added that because its a required prop, that would essentially be searching `TypeLink`, but why not, sure!

Reply:
i do need an assertion for `onClickField` thoough